index.html

```diff
@@ -50,127 +55,148 @@ <h1>I’m Mohammed Glad to see you!</h1>
       </ul>
     </div>
   </section>
+
   <section class="work-section">
     <div class="card1 card-work">
-      <img src="./img/Snapshoot-Portfolio.svg" alt="" />
-      <h1>Tonic</h1>
-      <div class="frame2">
-        <p>CANOPY</p>
-        <ul>
-          <li>Back End Dev</li>
-          <li>2015</li>
+      <div class="card-work__img">
+        <img src="./img/Snapshoot-Portfolio.svg" alt="" />
+      </div>
+      <div class="card-work__info">
+        <h1>Tonic</h1>
```

Comment:
Kindly change the `h1` tag in each project to be any heading between `h2` and `h6`, for example: `h2`. since the `h1` tag should only exist once on your page.